br/pkg/lightning/restore/tidb.go

```diff
@@ -100,6 +100,13 @@ func DBFromConfig(dsn config.DBStore) (*sql.DB, error) {
 			"autocommit": "1",
 		},
 	}
+
+	if dsn.Vars != nil {
```

Comment:
how can we test this change (by DM's CI in future?)

Reply:
I will add this test in DM when the PR is merged to master.